backend/consultations/signals.py

```
@@ -0,0 +1,14 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .models import Consultation
+from .emails import send_new_consultation_alert, send_response_notification
+
+@receiver(post_save, sender=Consultation)
+def consultation_post_save(sender, instance, created, **kwargs):
```

Ruff (0.15.0)

[warning] Unused function argument: `sender`

(ARG001)